homeassistant/components/light/tradfri.py

```diff
@@ -299,9 +299,14 @@ def _refresh(self, light):
 
         if self._light_data.hex_color is not None:
             if self._light.device_info.manufacturer == IKEA:
-                self._features |= SUPPORT_COLOR_TEMP
+                if "CWS" in self._light.device_info.model_number:
```

Comment:
As far as what I'm seeing in my Tradfri bulbs, `WS` means White Spectrum, and `CWS` means Colour and White Spectrum, therefore I suggest set supported features for IKEA bulbs regardless of `hex_color`, as in #9703:
```
        if self._light.device_info.manufacturer == IKEA:
            if "WS" in self._light.device_info.model_number:
                  self._features |= SUPPORT_COLOR_TEMP
            if "CWS" in self._light.device_info.model_number:
                self._features |= SUPPORT_RGB_COLOR
        else:
            if self._light_data.hex_color is not None:
                  self._features |= SUPPORT_RGB_COLOR
```

Reply:
Point taken. 
Both string comparisons will match for IKEA CWS/RGB bulbs which is fine IMHO. 
Color temp and RGB feature assumed for other bulbs whenever hex value is found.  

Also woof!